docs/scos/dev/front-end-development/202307.0/oryx/technology.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Oryx: Technology"
+description: The technologies Oryx is built on
+template: concept-topic-template
+last_updated: Oct 23, 2023
+---
+
+Oryx is a frontend framework designed to help developers build web applications using modern technologies and leveraging [Spryker API](https://docs.spryker.com/docs/scos/dev/glue-api-guides/{{page.version}}/decoupled-glue-api.html).
+
+Oryx leverages standard technology, frameworks and tools provided by the web platform and the community wherever possible.  It emphasizes utilizing the capabilities of the web and minimizing the reliance on build tools. During development, Oryx primarily employs JavaScript (JS) but aims to reduce the need for executing JS in the browser. It prioritizes HTML and CSS for delivering a fast-loading experience across various screen sizes.
```

Comment:
Is `wherever possible` related to only `the community` or to all the items in the sentence?

Reply:
To all items.